pyogrio/tests/test_raw_io.py

```diff
@@ -363,3 +363,47 @@ def test_read_from_file_like(tmpdir, naturalearth_lowres, driver, ext):
         result2 = read(f)
 
     assert_equal_result((meta, index, geometry, field_data), result2)
+
+
+@pytest.mark.parametrize(
+    "driver,ext", [("GPKG", "gpkg"), ("FlatGeobuf", "fbg")]
+)
+def test_read_write_data_types_numeric(tmp_path, driver, ext):
+    # Point(0, 0), Point(1, 1), Point(2, 2)
+    geometry = np.array([
```

Comment:
To keep this more readable, I'd suggest just adding a convert step from the pygeos geometries to WKB.

Reply:
For now I kept the raw WKB, but I simplified it a bit (we are not really using the geometries anyway in the actual test, it are just some dummy values). 
I could also have moved the whole test to test_geopandas.py, but since it is here now in raw IO and with the simplified code using pygeos would not reduce the lines of code, I would say it is fine to not use pygeos here.